### PR TITLE
Added handlebars for landing page and venue routes.

### PR DESCRIPTION
**Pull request by tejasraibagi26**
**Created handlebars for landing page and venue.**
1. Landing page has a form with textfield to input a searchTerm or simple search to fetch all venues.
2. Venue page display venue according to the searchTerm
3. Added cost filter
4. Individual venue renders specific venue with details to book slot timings

### **Landing Page**
Screen Shot 2021-11-10 at 4 00 31 PM

### **Search All**
Screen Shot 2021-11-10 at 4 00 41 PM

### **Search Specific**
Screen Shot 2021-11-10 at 4 00 56 PM

### **Search specific with cost filter** (check url)
Screen Shot 2021-11-10 at 4 01 10 PM

### **Individual Venue**
Screen Shot 2021-11-10 at 4 01 16 PM

### CODE REVIEW

**Review by troytomasch — CHANGES_REQUESTED** [review]

File: backup.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<section id="search-results">
```
[the remark this answers is not in the document]
Comment (tejasraibagi26) [reply backup.txt:1]: Oh my bad this wasn't supposed to be here. This is just a backup code of a file as I was testing multiple design choices.

File: routes/request.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    const addReq = await requestData.postRequest(
+      posterId,
+      venueId,
+      requestText,
+      date,
+      time
+    );
+    res.status(200).json(addReq);
+  } catch (error) {
+    res.status(404).json({ err: error });
```
[the remark this answers is not in the document]
Comment (tejasraibagi26) [reply routes/request.js]: I've made these changes and also added couple more handlebars. Please check.

**Review by troytomasch — COMMENTED** [review]
Just a couple print statements I saw. Otherwise, I'm good with it

File: data/venue.js [hunk 3]
```diff
@@ -60,27 +77,42 @@ const createNewVenue = async (venueName, venueAddress, venueTimings) => {
...
   if (createVenue.insertedCount === 0) throw "Error inserting venue";
 
-  return { msg: "Venue added!" };
+  return createVenue.insertedId;
 };
 
-const searchVenue = async (venueName) => {
-  let array = [venueName];
+const searchVenue = async (sportToFind, min, max, rating) => {
+  console.log(sportToFind);
```
[the remark this answers is not in the document]
Comment (tejasraibagi26) [reply data/venue.js:84]: Oh look's like i forgot to remove the console logs. I'll make sure to remove all of them in my next PR. Im ahead of the current PR so can't push the changes now or most of the code will be half baked.

**Comment by tejasraibagi26** [comment]
Go ahead and merge the it.

**Pull request merged**